_sass/settings/_variables.sass

```diff
@@ -19,7 +19,7 @@ $v-t-sans--light: 'Larsseit-Light', Helvetica, Arial sans-serif, sans-serif
 $v-t-sans--bold: 'Larsseit-Bold', Helvetica, Arial sans-serif, sans-serif
 
 // Units
-$v-t-size--base: 12px
+$v-t-size--base: 16px
```

Comment:
I'm pretty sure it's all good, but Allegra will flag it if font sizes are out of whack